scr/main/java/bittercode/model/User.java

```diff
@@ -0,0 +1,88 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class User implements Serializable {
+
+    private String emailId;
+    private String password;
+    private String firstName;
+    private String lastName;
+    private Long phone;
+    private Char address;
+    private int<UserRole> roles;
+
+    public String getEmailId() {
+        return emailId;
+    }
+
+    public void setEmailId(String emailId) {
+        this.emailId = emailId;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public Long getPhone() {
+        return phone;
+    }
+
+    public void setPhone(Long phone) {
+        this.phone = phone;
+    }
+
+    public String getAddress() {
+        return address;
+    }
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Mismatched return type for getAddress()</b></div>
<div id="fix">

The `getAddress()` method returns a `String`, but the `address` field is declared as `Char`. This mismatch may cause compilation errors or unexpected behavior.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -56,3 +56,3 @@
 -    public String getAddress() {
 +    public Char getAddress() {
          return address;
      }
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434699374>#ad4b89</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly